client-app/core/api/graphql/catalog/queries/getProductWishlistIds/index.ts

```diff
@@ -0,0 +1,21 @@
+import { globals } from "@/core/globals";
+import { graphqlClient } from "../../../client";
+import getProductWishlistIdsDocument from "./getProductWishlistIdsQuery.graphql";
+import type { Query, QueryProductArgs } from "@/core/api/graphql/types";
```

Comment:
use GetWishlistsProductQuery 
example in 
client-app\core\api\graphql\store\queries\getStore\index.ts